src/clj/web/api.clj

```diff
@@ -29,6 +30,10 @@
            (POST "/logout" [] auth/logout-handler)
            (GET "/check/:username" [] auth/check-username-handler)
            (PUT "/profile" [] auth/update-profile-handler)
+           (GET "/user" [] data/get-user)
+
+           (POST "/user/clearstats" [] stats/clear-user-stats)
+           (POST "/data/decks/clearstats" [] stats/clear-deck-stats)
```

Comment:
Let's make this a DELETE route on "/profile/stats/decks".

Reply:
A-OK